addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -99,7 +123,7 @@ export class SearchBar extends Component {
         }
 
         for (const searchItem of this.searchItems) {
-            const field = this.fields[searchItem.fieldName];
+            const field = this.fields[searchItem.fieldName.split(".")[0]];
```

Comment:
so in the end searchitem.fieldName is not a fieldName... I don't like this. Not natural